gateway/utils/graphql/graphql.go

```diff
@@ -208,42 +208,15 @@ func (graph *Module) execGraphQLDocument(ctx context.Context, node ast.Node, tok
 
 		currentValue, err := utils.LoadValue(fmt.Sprintf("%s.%s", store["coreParentKey"], field.Name.Value), store)
 		if err != nil {
-			cb(nil, nil)
+			cb(nil, nil) // todo should we return error here
```

Comment:
This will be nil. Add a comment on top saying that if the field isn't found in the state means that field did not exist in the result. In that case, we simply return nil. However no error really exists so we do not return an error